pkg/server/plugin/datastore/sql/sql.go

```diff
@@ -1150,7 +1163,13 @@ func modelToBundle(model *Bundle) (*datastore.Bundle, error) {
 	return bundle, nil
 }
 
+// validateRegistrationEntry validate the registration entry is non-nil, has >0 selectors,
```

Comment:
Considering this is a non-doc comment, it seems to provide too much detail and risks becoming stale if the function internals change. Validation details seem easy to discern from the function body, which is small and simple.

I'd suggest removing the comment.